src/cljs/org/broadinstitute/firecloud_ui/page/library/library_page.cljs

```diff
@@ -65,14 +66,20 @@
        (when-not (= current-search-text new-search-text)
          (react/call :update-query-params (@refs "table") {:filter-text new-search-text}))))
    :check-access
-   (fn [data props]
+   (fn [{:keys [state]} data props]
+     (utils/cljslog data)
+     (utils/cljslog props)
      (endpoints/call-ajax-orch
-       {:endpoint (endpoints/check-bucket-read-access {:namespace (:namespace data) :name (:name data)})
+       {:endpoint (endpoints/get-workspace {:namespace (:namespace data) :name (:name data)})
```

Comment:
Maybe ugly, but this should work as just `(endpoints/get-workspace data)`.  Maybe create a `(defn row->ws-id [row] (select-keys row [:namespace :name]))` function to be a little clearer.